translations/base-tr.yaml

```diff
@@ -658,32 +666,40 @@ storyRewards:
         desc:
             "Bu seviyenin bir ödülü yok ama bir sonrakinin olacak!<br><br> Not: Şu anki fabrikalarını yok etmemeni öneririm
             - Daha sonra <strong>Geliştirmeleri açmak için </strong> <strong>bütün hepsine</strong> ihtiyacın olacak!"
+
     no_reward_freeplay:
         title: Sonrakİ Sevİye
         desc: Tebrikler!
     reward_freeplay:
         title: Özgür Mod
+
         desc: Başardın! <strong>Özgür modu</strong> açtın! Bu artık gelen şekillerin
             <strong>rastgele</strong> oluşacağı anlamına geliyor!<br><br>
             Bundan sonra ana bölge belirli bir miktar eşya değil <strong>belirli bir miktar eşya geliş hızına</strong>
             bağlı olarak level atlayacaksın, istenilen şekilleri otomatik olarak yapacak bir fabrika inşa etmeni
             öneririm!<br><br> Ana bölgenin istediği şekil kablo katmanında sol taraftan sinyal olarak gönderiliyor,
             yani sadece bu şekli analiz ederek üretecek tamen otomatik bir alet yapman yeterli.
+
     reward_demo_end:
         title: Deneme Sürümünün Sonu
         desc: Deneme sürümünün sonuna geldin!
     reward_balancer:
         title: Dengeleyici
-        desc: Çok işlevli - bütün girdileri eşit olarak bütün çıkışlara dağıtır.!<br><br>
+
+        desc:  Çok fonksioynlu <strong>dengeleyeliyiciyi</strong> açtın!! - daha büyük 
+        fabrikalar yaratmmak için <strong>eşyaları</strong> birden çok konveyorlara ayırıp 
+        birleştirmek için kullanılabilir!<br><br>
     reward_merger:
         title: Tekil Birleştirici
-        desc: İki taşıma bandını bir çıktı verecek şekilde birleştirir!
+        desc: <strong>Birleştiriciyi</strong> açtın ! 
+            <strong>dengeleyecinin</strong> bir türü - İki giriş alıp tek banta atar.
+
     reward_belt_reader:
         title: Bant Okuyucu
         desc: <strong>Bant okuyucu</strong> açıldı! Bu yapı taşıma bandındaki akış
             hızını ölçmeyi sağlar.<br><br>Kabloları açana kadar bekle - o zaman
             çok kullanışlı olacak.
-    reward_rotater_180:
```

Comment:
This key is still used by game, therefore it should not be deleted. It is part of the game and it will result error while running the game.